spec/ddoc.dd

```diff
@@ -834,6 +834,10 @@ $(P
     $(TROW $(ARGS $(D DDOC_KEYWORD)), $(ARGS Highlighting of D keywords.))
     $(TROW $(ARGS $(D DDOC_PARAM)), $(ARGS Highlighting of function parameters.))
     $(TROW $(ARGS $(D DDOC_BACKQUOTED)), $(ARGS Inserts inline code.))
+    $(TROW $(ARGS $(D DDOC_AUTO_PSYMBOL)), $(ARGS Automatic highlighting of declaration name to which a particular section is referring in running text.))
+    $(TROW $(ARGS $(D DDOC_AUTO_KEYWORD)), $(ARGS Automatic highlighting of D keywords in running text.))
+    $(TROW $(ARGS $(D DDOC_AUTO_PARAM)), $(ARGS Automatic highlighting of declaration name to which a particular section is referring in running text.))
```

Comment:
This only happens in Ddoc comments - not in pure DD files, so maybe clarify as "in documentation text" or "in documentation header of a symbol"?